asteroidsyncserviced/platforms/ubuntutouch/main.cpp

```diff
@@ -0,0 +1,17 @@
+#include <QCoreApplication>
+#include <QDebug>
+#include "../../dbusinterface.h"
+#include "../../watchesmanager.h"
+#include "ubuntuplatform.h"
```

Comment:
Could you keep a shared main.cpp that has its #include "xxplatform.h" and object creation inside an ifdef block?

Reply:
I have to see if I can do this

Reply:
I made a first try with ifdef block defined in the *.pri files
```suggestion
    #if UBUNTU_TOUCH_PLATFORM
      UbuntuPlatform *platform = new UbuntuPlatform(watchesManager);
    #endif    
    #if SAILFISHOS_PLATFORM
      SailfishPlatform *platform = new SailfishPlatform(watchesManager);
    #endif
```